### ISSUE TITLE
Storefinder a11y e2e test fail

### ISSUE THREAD

**Issue by mirkosoft** [SAP/spartacus#6998]
Run a11y storefinder e2e tests against dev17:
`yarn e2e:cy:run:ci -s cypress/integration/accessibility/tabbing-order.e2e-spec.ts`
and see one of them fails:
![image](https://user-images.githubusercontent.com/788092/77179133-5a58e480-6ac8-11ea-8498-ccdecb5d4488.png)

**Comment by Matejk00** [comment]
QA steps:

Run all storefinder a11y e2e tests from `cypress\integration\accessibility\tabbing-order.e2e-spec.ts` and check if all of them are passing.

**Comment by mirkosoft** [comment]
Server QA passed.
